forge/ee/routes/flowBlueprints/index.js

```diff
@@ -4,7 +4,12 @@ module.exports = async function (app) {
     app.config.features.register('flowBlueprints', true, true)
 
     app.get('/', {
-        preHandler: app.needsPermission('flow-blueprint:list'),
+        preHandler: async (request, reply) => {
+            if (request.session.ownerType === 'project' || request.session.ownerType === 'device') {
```

Comment:
could devices and instances access this endpoint?

Reply:
Devices and Instances are accessing this endpoint using the new BluePrint library plugin -> https://github.com/FlowFuse/nr-launcher/pull/345